### DISCUSSION TITLE
Are artifacts links no longer available for unauthorised users?

### DISCUSSION THREAD

**Discussion by alexeylopukh-aloteq** [codemagic-ci-cd/codemagic-docs#1371] [Q&A] [answered]
For me, the download is available, but if I share it with someone or open it in incognito mode, I get an error:
`{"error":"FORBIDDEN","message":"The server could not verify that you are authorized to access the URL requested. You either supplied the wrong credentials (e.g. a bad password), or your browser doesn't understand how to supply the credentials required."}`

They used to be available to everyone.
Example link: https://api.codemagic.io/artifacts/example/example.apk

**Comment by himesh-cm** [comment, marked as answer]
Hi @alexeylopukh-aloteq, The artifact links require authentication now; the links in email/slack are valid for 24h.
You can read more about the change [here](https://docs.codemagic.io/notes/release-notes/#artifact-url-expiration)
